efficientdet/hparams_config.py

```diff
@@ -161,6 +161,8 @@ def as_dict(self):
       else:
         config_dict[k] = copy.deepcopy(v)
     return config_dict
+
+
 # pylint: enable=protected-access
 
```

Comment:
Maybe you can move "# pylint: enable=protected-access" right after return (with same indent), to avoid too many empty lines.